docker/statutory/Dockerfile

```diff
@@ -20,9 +22,8 @@ USER node
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH="/home/node/.npm-global/bin:${PATH}"
 
-RUN npm install -g --loglevel warn nodemon bunyan && npm cache clean --force
-RUN npm install --loglevel warn
+RUN npm install
 
-CMD sh /usr/app/scripts/bootstrap.sh && nodemon -e "js,json" lib/run.js
+CMD ["sh /usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]
```

Comment:
isn't it the syntax where you pass the command and its arguments separately? like `CMD ["sh", "/usr/app/scripts/bootstrap.sh", ...]`

Reply:
That might be the case, I will check